omnigibson/utils/bddl_utils.py

```diff
@@ -89,9 +127,810 @@ def process_single_condition(condition):
     "open": get_unary_predicate_for_state(object_states.Open, "open"),
     "toggled_on": get_unary_predicate_for_state(object_states.ToggledOn, "toggled_on"),
     "frozen": get_unary_predicate_for_state(object_states.Frozen, "frozen"),
+    "future": ObjectStateFuturePredicate,
+    "real": ObjectStateRealPredicate,
 }
 
+KINEMATIC_STATES_BDDL = frozenset([state.__name__.lower() for state in _KINEMATIC_STATE_SET])
+
+# Load substance and object mapping
+with open(f"{bddl.__path__[0]}/../substance_synset_mapping.json", "r") as f:
+    SUBSTANCE_SYNSET_MAPPING = json.load(f)
+
+
+# BEHAVIOR-related
+OBJECT_TAXONOMY = ObjectTaxonomy() #hierarchy_type="b1k")
+FLOOR_SYNSET = "floor.n.01"
```

Comment:
let's get rid of `FLOOR_SYNSET`

Reply:
what should this be? will floor automatically be written into non_sampleable categories? `NON_SAMPLEABLE_SYNSETS = set([FLOOR_SYNSET] + [line.strip() for line in f.readlines()])`

Reply:
Yes, can we add floor to the list?

Reply:
added TODO